content/en/docs/concepts/otel-concepts.md

```diff
@@ -0,0 +1,392 @@
+- **measure**: a value that is aggregated over time. This is more akin to the
+  trip odometer on a car, it represents a value over some defined range.
+- **observer**: captures a current set of values at a particular point in time,
+  like a fuel gauge in a vehicle.
+
+In addition to the three metric instruments, the concept of _aggregations_ is an
+important one to understand. An aggregation is a technique whereby a large
+number of measurements are combined into either exact or estimated statistics
+about metric events that took place during a time window. The OpenTelemetry API
+itself does not allow you to specify these aggregations, but provides some
+default ones. In general, the OpenTelemetry SDK provides for common aggregations
+(such as sum, count, last value, and histograms) that are supported by
+visualizers and telemetry backends.
+
+Unlike request tracing, which is intended to capture request lifecycles and
+provide context to the individual pieces of a request, metrics are intended to
+provide statistical information in aggregate. Some examples of use cases for
+metrics include:
+
+- Reporting the total number of bytes read by a service, per protocol type.
+- Reporting the total number of bytes read and the bytes per request.
+- Reporting the duration of a system call.
+- Reporting request sizes in order to determine a trend.
+- Reporting CPU or memory usage of a process.
+- Reporting average balance values from an account.
+- Reporting current active requests being handled.
+
+> For more information, see the [metrics specification][].
+
+## Logs in OpenTelemetry
+
+A **log** is a timestamped text record, either structured (recommended) or
+unstructured, with metadata. While logs are an independent data source, they may
+also be attached to spans. In OpenTelemetry, any data that is not part of a
+distributed trace or a metric is a log. For example, _events_ are a specific
+type of log. Logs are often used to determine the root cause of an issue and
+typically contain information about who changed what as well as the result of
+the change.
+
+> For more information, see the [logs specification][].
+
+## Baggage in OpenTelemetry
+
+Imagine you want to have a `CustomerId` attribute on every span in your trace,
+which involves multiple services; however, `CustomerId` is only available in one
+specific service. To accomplish your goal, you can use OpenTelemetry Baggage to
+propagate this value across your system.
+
+In OpenTelemetry, "Baggage" refers to contextual information that’s passed
+between spans. It's a key-value store that resides within a trace context,
+making values available to any span created within that trace.
+
+OpenTelemetry uses [Context Propagation](#context-propagation) to pass Baggage
+around, and each of the different library implementations has propagators that
+will parse and make that Baggage available without you needing to explicitly
+implement it.
+
+![OTel Baggage](/img/otel_baggage.png)
```

Comment:
@cartermp does this diagram illustrate baggage a bit better? If not let me know what would - open to suggestions!

Reply:
It does, thank you!